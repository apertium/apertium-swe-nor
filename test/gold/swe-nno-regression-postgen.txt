[4ezRqmPuJ3lS]
har vorte oppført [/option]
[/4ezRqmPuJ3lS]
[BGSXM-VaHxKz]
som ikkje har vorte oppført [/option]
[/BGSXM-VaHxKz]
[BcOHchZgEltn]
meir overskodeleg. [/option]
[/BcOHchZgEltn]
[EM8lSAP-warx]
Eg fekk òg ei ny kusine i år som eg nesten ikkje har sett. [/option]
[/EM8lSAP-warx]
[Ep5NrTWjxfXF]
Tittel: Anna får ikkje reisa frå byen ho bur i. [/option]
[/Ep5NrTWjxfXF]
[FGbYle80JvRe]
han. [/option]
[/FGbYle80JvRe]
[I1a7gtD3zHRi]
har funnest [/option]
[/I1a7gtD3zHRi]
[URVJjX_lB38B]
mest overskodelege. [/option]
[/URVJjX_lB38B]
[V01qA6qi0kKt]
Ingress: Anna er bare ihop med vennene sine på Teams no for tida. [/option]
[/V01qA6qi0kKt]
[W-fqMeU--uw4]
som har funnest [/option]
[/W-fqMeU--uw4]
[cN2TnhvQvwvm]
mest overskodeleg. [/option]
[/cN2TnhvQvwvm]
[dMemZE94Geg0]
Pannekaker og lasagne. [/option]
[/dMemZE94Geg0]
[eLrB2F8Y26uc]
søkjande der evna til å utnytta [/option]
[/eLrB2F8Y26uc]
[gqecOW-8qUFZ]
har vorte oppført. [/option]
[/gqecOW-8qUFZ]
[igT2HXzyiwds]
har funnest. [/option]
[/igT2HXzyiwds]
[jONRw3Es7Fmy]
dei partane som har søknader som [/option]
[/jONRw3Es7Fmy]
[w0tTxQc12qvl]
Såg ho ikkje orda? [/option]
[/w0tTxQc12qvl]
[xr87sIaNgHKM]
Tsjekkia har vorte ramma hardt av koronaviruset. [/option]
[/xr87sIaNgHKM]
[yaT9Y83vKE5G]
Det har vore trist! [/option]
[/yaT9Y83vKE5G]
[zTuZzFUZoMgA]
han [/option]
[/zTuZzFUZoMgA]
