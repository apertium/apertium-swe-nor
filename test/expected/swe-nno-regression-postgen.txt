[0tkleIoBBRxj#0] mest #overskuelig
[/0tkleIoBBRxj]
[4ezRqmPuJ3lS#0] har vorte oppført
[/4ezRqmPuJ3lS]
[BGSXM-VaHxKz#0] som ikkje har vorte oppført
[/BGSXM-VaHxKz]
[I1a7gtD3zHRi#0] har vorte funne
[/I1a7gtD3zHRi]
[W-fqMeU--uw4#0] som har vorte funne
[/W-fqMeU--uw4]
[eLrB2F8Y26uc#0] #søkje der evna til å utnytta\/utnytte
[/eLrB2F8Y26uc]
[hqvcBLgqrUlV#0] mest #overskuelig
[/hqvcBLgqrUlV]
[i9_os4amzrP1#0] meir #overskuelig
[/i9_os4amzrP1]
[jONRw3Es7Fmy#0] dei partane som har søknader som
[/jONRw3Es7Fmy]
[w0tTxQc12qvl#0] Såg ho ikkje orda?
[/w0tTxQc12qvl]
[zTuZzFUZoMgA#0] han
[/zTuZzFUZoMgA]
